ElectronApp/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+
+import '@testing-library/jest-dom';
+
+import App from '../renderer/App';
+import Nav from '../renderer/Nav';
+
+describe('App', () => {
+  it('should render', () => {
+    expect(render(<App />, { wrapper: MemoryRouter })).toBeTruthy();
+  });
+});
+
+describe('Nav', () => {
```

Comment:
Since these tests are now describing the Nav, not the App, I think it makes sense to place them in a separate file called `Nav.test.tsx`